nuclia/sdk/auth.py

```diff
@@ -184,33 +191,29 @@ def post_login(self):
         self.accounts()
         self.zones()
 
-    def post_user(self, path: str, payload: Dict[str, str]):
-        if not self._config.token:
-            raise NeedUserToken()
-        resp = requests.post(
-            path,
-            headers={"Authorization": f"Bearer {self._config.token}"},
-            data=payload,
-        )
-        if resp.status_code == 201:
-            return resp.json()
-        elif resp.status_code == 403:
-            raise UserTokenExpired()
-
-    def get_user(self, path: str):
+    def _request(self, method: str, path: str, data: Optional[Any] = None):
         if not self._config.token:
             raise NeedUserToken()
-        resp = requests.get(
+        kwargs = {"headers": {"Authorization": f"Bearer {self._config.token}"}}
+        if data is not None:
+            non_null_values = {k: v for k, v in data.items() if v is not None}
```

Comment:
Not sure if there is a use-case, but null/None values may be required at some point, for instance to reset a specific attribute to None.